src/common/common.gyp

```diff
@@ -0,0 +1,26 @@
+{
+  'includes':[
+    'common.gypi',
+  ],
+  'targets': [{
+    'target_name': 'tizen_common',
+    'type': 'static_library',
+    'sources': [
+      'extension.cc',
+      'extension.h',
+      'logger.cc',
+      'logger.h',
+      'picojson.h',
+      'scope_exit.h',
+      'utils.h',
+      'XW_Extension.h',
+      'XW_Extension_EntryPoints.h',
+      'XW_Extension_Permissions.h',
+      'XW_Extension_Runtime.h',
+      'XW_Extension_SyncMessage.h',
+    ],
```

Comment:
I'm sorry, but not quite understand. Do you mean that we should place -std=c++0x flag in common.gyp?
It's already defined in common.gypi which is also included in common.gyp.